README.md

```diff
@@ -1,21 +1,44 @@
 # Destiny - Cross-platform Magic Wormhole graphical client
 
-⚠ This project is still a work in progress, use at your own risk ⚠
+## An end-to-end encrypted app to share files without revealing your identity 
 
-Easily send and receive files without signup or loss of privacy. Based on the
-[wormhole-william](https://github.com/LeastAuthority/wormhole-william)
+Destiny is a secure file transfer application that allows people to transfer files without needing to reveal their identity to each other or the service provider. All files are end-to-end encrypted, meaning no one except the sender and the receiver we cannot see their contents. Users select a file on their device then share the generated code with the intended recipient for safe delivery. No sign-up needed.
+
+Key security features: 
+
+- **Identity-less**: no need to disclose identity information (such as name, email address or phone number) to be able to transfer files.
+- **End-to-end encryption**: files are encrypted-at-rest and only the sender and recipient can decrypt them.
+
```

Comment:
Here is the exact text from the FAQ to use instead. Meejah was saying that "encrypted at rest" is not the best phrase to use. see below + other key features listed 


Identity-less: no need to disclose identity information (such as name, email address or phone number) to be able to transfer files.
End-to-end encryption: files are end-to-end encrypted and only the sender and recipient can read them.
Peer-to-peer file sharing: Destiny attempts to make a direct network connection to the other party. If both parties are on the same local network they should connect without any traffic leaving that network, for example. When this isn’t possible (e.g. if neither party has a public IP address) then our relay server is used (however, that server sees only encrypted packets).
Full-strength keys: although our codes are short and human-memorable, they are part of an online “Password Authenticated Key Exchange” (PAKE) which only allows a single guess – and yields a 256-bit full-strength symmetric key.

Reply:
Fixed